heft-plugins/heft-webpack5-plugin/src/shared.ts

```diff
@@ -36,12 +48,9 @@ export interface IWebpackBundleSubstageProperties extends IBundleSubstagePropert
    */
   // We are inheriting this problem from Tapable's API
```

Comment:
Problem?

Reply:
Yes, returning `undefined` on a waterfall hook means "keep the current value and move on to the next tap". However, we also want to allow developers to override/set the configuration value to undefined (to signal that Webpack shouldn't be run at all).

I'll change it from null to `false` to work around this without needing any special eslint ignores.